templates/teedy-docs.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>Teedy-docs</Name>
+  <Repository>sismics/docs:latest</Repository>
+  <Registry>https://hub.docker.com/r/sismics/docs/</Registry>
+  <Branch>
+  <Tag>latest</Tag>
+  <TagDescription>Master branch, can be unstable. Not recommended for production use</TagDescription>
+  </Branch>
+  <Branch>
+  <Tag>v1.8</Tag>
+  <TagDescription>Latest stabel version</TagDescription>
+  </Branch>
+  <Network>bridge</Network>
+  <Privileged>false</Privileged>
+  <Support>https://hub.docker.com/r/sismics/docs/</Support>
+  <Project>https://github.com/sismics/docs</Project>
+  <Overview>Teedy is an open-source, lightweight document management system (DMS) for individuals and businesses.&#xD;
+&#xD;
+The default admin password is "admin". Don't forget to change it before going to production.</Overview>
+  <Category>Cloud: Productivity:</Category>
+  <WebUI>http://[IP]:[PORT:8080]</WebUI>
+  <Icon>https://github.com/sismics/docs/blob/master/docs-web/src/main/resources/image/logo.png?raw=true</Icon>
```

Comment:
The actual link: https://raw.githubusercontent.com/sismics/docs/master/docs-web/src/main/resources/image/logo.png

We usually host the image on our repo, in case the one on the project gets moved.

Reply:
Fixed to the actual link, thank you :)